py/src/ai/h2o/sparkling/ml/params/H2OGridSearchParams.py

```diff
@@ -124,22 +138,24 @@ def getSelectBestModelBy(self):
         return self.getOrDefault(self.selectBestModelBy)
 
     def getSelectBestModelDecreasing(self):
-        return self.getOrDefault(self.selectBestModelDecreasing)
+        Utils.deprecationWarning("getSelectBestModelDecreasing", "getSelectBestModelOrdering")
+        ordering = self.getOrDefault(self.selectBestModelOrdering)
+        return ordering is "ASC"  # The default value was DESC as well
```

Comment:
`ordering is "DESC"`?

Reply:
Good catch! This won't be needed at the end as the option will just get deprecated without replacement in a form of `selectBestModelOrdering`